### ISSUE TITLE
Doesn't fetch new wallpaper automatically

### ISSUE THREAD

**Issue by SahajPatel** [ifl0w/RandomWallpaperGnome3#76]
Does not fetch new wallpaper by itself on Unsplash for me, running Pop!_OS 19.10, Gnome is on version 3.34.1

**Comment by pmav** [comment]
I'm having the same issue.

Anything I could do to help? Logs, debugging, etc.

**Comment by pmav** [comment]
Error log:

```
Nov 08 22:07:21 paradox gnome-shell[2028]: RWG3 [INFO] >> WallpaperController :: Requesting image: https://images.unsplash.com/photo-1571116887105-e99e0b6079a9?ixlib=rb-1.2.1&q=85&fm=jpg&crop=entropy&cs=srgb
Nov 08 22:07:21 paradox gnome-shell[2028]: RWG3 [ERROR] >> WallpaperController :: Could not load new wallpaper. (Gio.IOErrorEnum: Operation not supported)
Nov 08 22:07:21 paradox gnome-shell[2028]: JS ERROR: TypeError: tmp.setIndex is not a function
                                           updateList@/home/pmav/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/elements.js:322:5
                                           setHistoryList@/home/pmav/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:177:3
                                           fetchNewWallpaper/</</<@/home/pmav/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/wallpaperController.js:245:44
                                           fetchNewWallpaper/</<@/home/pmav/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/wallpaperController.js:245:6
                                           _fetchFile/<@/home/pmav/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/wallpaperController.js:136:6
```

**Comment by hamomel** [comment]
Also have this issue. The logs are the same.
Fedora 31 
Gnome Version 3.34.1

**Comment by leonardehrenfried** [comment]
This is because the extension has some issues in Gnome 3.34: #72

**Comment by ifl0w** [comment]
This should be resolved as well thanks to #77.
